evadb/executor/create_function_executor.py

```diff
@@ -674,7 +682,10 @@ def exec(self, *args, **kwargs):
             msg = f"Function {self.node.name} overwritten."
         else:
             msg = f"Function {self.node.name} added to the database."
-        yield Batch(pd.DataFrame([msg]))
+        if best_score and train_time:
+            yield Batch(pd.DataFrame([msg, best_score, train_time]))
```

Comment:
Could we add some text? Other wise, it will be just two numbers without any explanation.

Reply:
Added some annotations